openpype/hosts/houdini/api/lib.py

```diff
@@ -1054,3 +1054,72 @@ def add_self_publish_button(node):
     template = node.parmTemplateGroup()
     template.insertBefore((0,), button_parm)
     node.setParmTemplateGroup(template)
+
+
+def getSceneViewer():
```

Comment:
Why camel case?

Reply:
This should indeed be `get_scene_viewer()`
```suggestion
def get_scene_viewer():
```

Reply:
> Why camel case?

I did some copy pasting and forgot. 😁